pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -412,6 +412,10 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(pInfo *framework.QueuedPodI
 	} else {
 		p.unschedulableQ.addOrUpdate(pInfo)
 		metrics.SchedulerQueueIncomingPods.WithLabelValues("unschedulable", ScheduleAttemptFailure).Inc()
+		var val = pInfo.UnschedulablePlugins.List()
```

Comment:
we should increment the metric in both cases, whether it is being inserted to backoff or unschedulable queue. So please move it outside after the if/else

Reply:
@ahg-g Added in review commit